### PR TITLE
Stop geocoding firms and migrate existing geo data to main office records

### PR DESCRIPTION
**Pull request by munckymagik**
This is a clean up PR to remove unused fields from firm and migrate data to offices.

### CODE REVIEW

File: db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    Firm.geocoded.includes(:offices).find_each do |f|
+      next unless f.offices.present?
+
+      f.main_office.update!(latitude: f.latitude,
+                            longitude: f.longitude)
+
+      update_count += 1
+    end
+
+    say "Migrated #{update_count} coordinates to main offices"
```
Comment (asmega) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]: will this be very useful when you run it on the server?

Comment (munckymagik) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]: Exactly. Better than silence.

Or are you subtly hinting that I've missed something?

Comment (asmega) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]: when you run this on the server the will say hello? have you managed to someone connect speakers to the server or pipe the audio to your local machine

Comment (munckymagik) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]: Oh! I see, LOL :)

No it's not [say](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/say.1.html) it's [say](http://apidock.com/rails/ActiveRecord/Migration/say/class#211-print-standard-looking-messages-during-migration)

We're running migrations on heroku which mean we can see the output.

Comment (munckymagik) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]:
```
$ be rake db:migrate
== 20151111132037 RemoveLatitudeAndLongitudeFromFirm: migrating ===============
-- Migrated 2796 coordinates to main offices
-- remove_column(:firms, :latitude)
   -> 0.0010s
-- remove_column(:firms, :longitude)
   -> 0.0005s
-- (!!) Changes have been made that require reindexing (run: rake firms:index)
== 20151111132037 RemoveLatitudeAndLongitudeFromFirm: migrated (4.4668s) ======

$ be rake db:rollback STEP=1
== 20151111132037 RemoveLatitudeAndLongitudeFromFirm: reverting ===============
-- add_column(:firms, :latitude, :float)
   -> 0.0010s
-- add_column(:firms, :longitude, :float)
   -> 0.0006s
-- Migrated 2796 coordinates to back to firms
-- (!!) Changes have been made that require reindexing (run: rake firms:index)
== 20151111132037 RemoveLatitudeAndLongitudeFromFirm: reverted (5.2052s) ======
```

Comment (asmega) [inline db/migrate/20151111132037_remove_latitude_and_longitude_from_firm.rb]: ah i see, it's a feature/method i was not aware of!
makes sense now

**Comment by asmega** [comment]
:+1:

**Comment by mconnell** [comment]
:+1:

**Comment by spencer** [comment]
:+1:

**Pull request merged**